app/features/settings/application/SettingsApplicationPanel.js

```diff
@@ -19,25 +20,46 @@ class SettingsApplicationPanel extends Component {
         this.props.tapEndSetTimer()
     }
 
+    _tappedSetMetric() {
+        this.props.tapSetDefaultMetric()
+    }
+
     // RENDER
 
     render() {
         if (Platform.OS === 'ios') {
```

Comment:
Instead of making it a new "card" can you have it in same card as the "Time to start new set"?